clients/unshaded/src/main/java/tachyon/client/table/AbstractTachyonRawTables.java

```diff
@@ -30,26 +35,34 @@
 @PublicApi
 public abstract class AbstractTachyonRawTables implements TachyonRawTablesCore {
   protected RawTableContext mContext;
+  protected TachyonFileSystem mTachyonFileSystem;
 
   protected AbstractTachyonRawTables() {
     mContext = RawTableContext.INSTANCE;
+    mTachyonFileSystem = TachyonFileSystem.TachyonFileSystemFactory.get();
```

Comment:
Sure, I've also created a ticket to address this in the rest of the codebase: https://tachyon.atlassian.net/browse/TACHYON-1316